### PR TITLE
Improve query speed active_users_aggregates

### PR DESCRIPTION
**Pull request by lucia-vargas-a**
Adding aggregate awareness to the Explore for active_users_aggregates is one of the improvements to query performance when querying from Looker.

This PR adds two aggregates:

    Current year data for dimensions used in existing Looks under Browsers / <browser_name> / Usage
    Current and last year data for common dimensions identified during the creation of active_users_aggregates

### CODE REVIEW

File: combined_browser_metrics/combined_browser_metrics.model.lkml [hunk 1]
```diff
@@ -9,23 +9,54 @@ explore: active_users_aggregates {
...
-  relationship: one_to_one
-  sql_on: ${active_users_aggregates.country} = ${countries.code} ;;
-}
+  join: countries {
+    type: left_outer
+    relationship: one_to_one
+    sql_on: ${active_users_aggregates.country} = ${countries.code} ;;
+  }
+
+  aggregate_table: rollup__active_users_aggregates_2022_usage {
```
Comment (lucia-vargas-a) [inline combined_browser_metrics/combined_browser_metrics.model.lkml:18]: For fields used in Looks created under Browsers / <browser_name> / Usage

File: combined_browser_metrics/combined_browser_metrics.model.lkml [hunk 2]
```diff
@@ -9,23 +9,54 @@ explore: active_users_aggregates {
...
+      ]
+    }
+    materialization: {
+      sql_trigger_value: SELECT CURRENT_DATE() ;;
+      increment_key: active_users_aggregates.submission_date
+      increment_offset: 1
+    }
+  }
+
+  aggregate_table: rollup__active_users_aggregates_2021_common {
```
Comment (lucia-vargas-a) [inline combined_browser_metrics/combined_browser_metrics.model.lkml:33]: For fields commonly used in queries, data since 2021.

**Review by ANich — APPROVED** [review]
Approved, but I'm not sure why we'd want to use a combination of aggregation techniques here. If these are the commonly used fields, shouldn't we just aggregate them from the upstream/base table?

**Comment by lucia-vargas-a** [comment]
> Approved, but I'm not sure why we'd want to use a combination of aggregation techniques here. If these are the commonly used fields, shouldn't we just aggregate them from the upstream/base table?

I guess this is part of our discussion and proposal of how and when to use each type of aggregate.

My thoughts:
Aggregating in Looker from `unified_metrics` is not feasible as the calculations and logic implemented in `active_user_aggregates` cannot all be replicated using aggregate awareness.
If there is any part that could be replicated, it would imply moving the logic and the data model into Looker, creating a bigger dependency and making it too difficult to query aggregated data from outside Looker.

The purpose of using aggregate awareness here is to improve the performance by generating a subset from `active_user_aggregates`, while keeping the calculations and logic in BigQuery. One of the positive feedback received about this aggregate table is that the structure in Looker reflects the structure in BigQuery, which makes it very clear for all users.

**Pull request merged**